src/lib_ccx/activity.c

```diff
@@ -135,6 +135,6 @@ void activity_header (void)
 		credits_shown = 1;
 		mprint ("CCExtractor %s, Carlos Fernandez Sanz, Volker Quetschke.\n", VERSION);
 		mprint ("Teletext portions taken from Petr Kutalek's telxcc\n");
-		mprint ("--------------------------------------------------------------------------\n");
+		// mprint ("--------------------------------------------------------------------------\n");
```

Comment:
Why is this line commented out?

Reply:
For proper formatting in the man pages. This was not looking good